assets/queries/ansible/aws/db_instance_publicly_accessible/query.rego

```diff
@@ -0,0 +1,59 @@
+package Cx
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    isAnsibleTrue(task["community.aws.rds_instance"].publicly_accessible)
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.{{community.aws.rds_instance}}.publicly_accessible", [task.name]),
+        "issueType": "IncorrectValue",
+        "keyExpectedValue": sprintf("module's parameter community.aws.rds_instance.publicly_accessible should be false in task: '%s'", [task.name]),
+        "keyActualValue": sprintf("module's parameter community.aws.rds_instance.publicly_accessible is true in task: '%s'", [task.name])
+    }
+}
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    isAnsibleTrue(task.rds.publicly_accessible)
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.rds.publicly_accessible", [task.name]),
+        "issueType": "IncorrectValue",
+        "keyExpectedValue": sprintf("module's parameter rds.publicly_accessible should be false in task: '%s'", [task.name]),
+        "keyActualValue": sprintf("module's parameter rds.publicly_accessible is true in task: '%s'", [task.name])
+    }
+}
```

Comment:
This resource doesn't exist, the doc for this module is no longer in the docs: https://docs.ansible.com/ansible/latest/rds_module.html

Reply:
You're right. Good catch. Removing this section and sample.